setup.cfg

```diff
@@ -224,6 +224,8 @@ filterwarnings =
     # https://github.com/h5py/h5py/issues/2243 for upstream issues about this warning being
     # emitted
     ignore:`product` is deprecated as of NumPy 1.25.0:DeprecationWarning
+    # sunpy IO skip here to prevent pytest breaking items downstream
+    ignore:.*module is deprecated, as it was designed for internal use
```

Comment:
Add the sunpy deprecation class here to make it clear it's ours?

Reply:
sunpy.util.exceptions.SunPyDeprecationWarning should work?